rust/agama-dbus-server/src/network/model.rs

```diff
@@ -408,6 +450,69 @@ pub struct LoopbackConnection {
 }
 
 #[derive(Debug, Default, PartialEq, Clone)]
+pub struct BondingConnection {
+    pub base: BaseConnection,
+    pub bonding: BondingConfig,
```

Comment:
@teclator what do you think about this extracting all these options into seperate structs.
The dbus interface of NM is quite simple here([see](https://networkmanager.dev/docs/api/latest/nm-settings-dbus.html)). It only uses a `a{ss}`. So we could also do it here with a simple `HashMap<String:String>`.

Reply:
changed to `HashMap<String:String>`